edu.gemini.lch.services/src/main/scala/edu/gemini/lch/services/impl/JSpOCCredentials.scala

```diff
@@ -6,10 +6,11 @@ import org.apache.commons.httpclient.UsernamePasswordCredentials
 import scala.collection.immutable.HashMap
 
 object JSpOCCredentials {
-  val gsUserName = System.getenv("GS_USERNAME")
-  val gsPassword = System.getenv("GS_PASSWORD")
-  val gnUserName = System.getenv("GN_USERNAME")
-  val gnPassword = System.getenv("GN_PASSWORD")
+  val gsUserName = Option(System.getenv("GS_USERNAME")).getOrElse("")
+  val gsPassword = Option(System.getenv("GS_PASSWORD")).getOrElse("")
+  val gnUserName = Option(System.getenv("GN_USERNAME")).getOrElse("")
+  val gnPassword = Option(System.getenv("GN_PASSWORD")).getOrElse("")
```

Comment:
Do we really want empty strings?

Reply:
Oh I didn't see your suggestion. I guess we should fail though I'm not sure what will handle that error

Reply:
Where are the environment variables specified?

Reply:
Agree, we should fail if we can't find the credentials.